content/en/security/code_security/troubleshooting/_index.md

```diff
@@ -241,7 +241,9 @@ SCA can be enabled on your running services using one of the following two metho
 
 To disable SCA, you must use the *same method* you used to enable SCA.
 
-**Note**: If you enabled SCA manually, you must disable it manually. You cannot disable it using the UI.
+{{< tabs >}}
+{{% tab "UI" %}}
+**Note**: If you enabled SCA through the `DD_APPSEC_SCA_ENABLED` environment variable you cannot disable it using the UI.
 
 To disable SCA through the UI, you can:
 
```

Comment:
The links need to be added to the tab section (links in the UI section are broken in the preview): https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown+Shortcode+Reference#Tabs